### DISCUSSION TITLE
Trace Compass Cloud missing from CDT Cloud online demo?

### DISCUSSION THREAD

**Discussion by ebugden** [eclipse-cdt-cloud/cdt-cloud#35] [Q&A] [answered]
I just opened the CDT Cloud online demo (Dec. 8th) with the intention of trying Trace Compass Cloud. The Trace Compass Cloud component is mentioned in the "Get Started" page (see screenshot below), but the component seems to be missing from this version of CDT Cloud (there is no Trace Viewer icon in the side bar or in the View list).

Is the Trace Viewer missing from this version of CDT Cloud?  

![Screenshot from 2023-12-08 10-16-51-annotated](https://github.com/eclipse-cdt-cloud/cdt-cloud/assets/28311615/42ea89c5-73ba-4949-ab35-236a939b50da)

**Comment by sdirix** [comment, marked as answer]
Hi!

Thanks for pointing out the missing Trace Compass Cloud feature in the
online CDT Cloud demo. Just checked it myself, and can confirm, the Trace
Viewer isn't there. Odd, since we just added support for it not long ago.

We're on it and will sort out what's gone wrong. I'll update you as soon as
we have it fixed.

Sorry for the inconvenience and thanks again for the heads-up!

Cheers,
Stefan

On Fri, Dec 8, 2023 at 7:30 PM ebugden ***@***.***> wrote:

> I just opened the CDT Cloud online demo (Dec. 8th) with the intention of
> trying Trace Compass Cloud. The Trace Compass Cloud component is mentioned
> in the "Get Started" page (see screenshot below), but the component seems
> to be missing from this version of CDT Cloud (there is no Trace Viewer icon
> in the side bar or in the View list).
>
> Is the Trace Viewer missing from this version of CDT Cloud?
>
> Screenshot.from.2023-12-08.10-16-51-annotated.png (view on web)
> <https://github.com/eclipse-cdt-cloud/cdt-cloud/assets/28311615/42ea89c5-73ba-4949-ab35-236a939b50da>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse-cdt-cloud/cdt-cloud/discussions/35>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACEU3YEY2SP5DHFBOWFDF4TYINME7AVCNFSM6AAAAABANBB4BCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEZTSNRQGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Stefan Dirix

Senior Software Architect
EclipseSource

Email: ***@***.***
Web: https://eclipsesource.com

EclipseSource GmbH
Messerschmittstrasse 4
80992 Muenchen

General Managers: Dr. Jonas Helming, Dr. Maximilian Koegel
Registered Office: Messerschmittstrasse 4, 80992 Muenchen,
Commercial Register Muenchen, HRB 191789

**Comment by sdirix** [comment]
Hello Everyone,

Great news! The issue with Trace Compass Cloud in the CDT Cloud Blueprint
is now fixed. You can once again access the Trace Viewer in the online 'Try
now' demo.

It turns out, during a recent Theia Cloud deployment, an outdated version
of the CDT Cloud Blueprint was accidentally used. We will take steps to
prevent similar issues in the future.

A big thank you for reporting this and helping us improve. Happy tracing!

Cheers,
Stefan

On Sun, Dec 10, 2023 at 2:59 PM Stefan Dirix ***@***.***>
wrote:

> Hi!
>
> Thanks for pointing out the missing Trace Compass Cloud feature in the
> online CDT Cloud demo. Just checked it myself, and can confirm, the Trace
> Viewer isn't there. Odd, since we just added support for it not long ago.
>
> We're on it and will sort out what's gone wrong. I'll update you as soon
> as we have it fixed.
>
> Sorry for the inconvenience and thanks again for the heads-up!
>
> Cheers,
> Stefan
>
> On Fri, Dec 8, 2023 at 7:30 PM ebugden ***@***.***> wrote:
>
>> I just opened the CDT Cloud online demo (Dec. 8th) with the intention of
>> trying Trace Compass Cloud. The Trace Compass Cloud component is mentioned
>> in the "Get Started" page (see screenshot below), but the component seems
>> to be missing from this version of CDT Cloud (there is no Trace Viewer icon
>> in the side bar or in the View list).
>>
>> Is the Trace Viewer missing from this version of CDT Cloud?
>>
>> Screenshot.from.2023-12-08.10-16-51-annotated.png (view on web)
>> <https://github.com/eclipse-cdt-cloud/cdt-cloud/assets/28311615/42ea89c5-73ba-4949-ab35-236a939b50da>
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/eclipse-cdt-cloud/cdt-cloud/discussions/35>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ACEU3YEY2SP5DHFBOWFDF4TYINME7AVCNFSM6AAAAABANBB4BCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEZTSNRQGA>
>> .
>> You are receiving this because you are subscribed to this thread.Message
>> ID: ***@***.***>
>>
>
>
> --
> Stefan Dirix
>
> Senior Software Architect
> EclipseSource
>
> Email: ***@***.***
> Web: https://eclipsesource.com
>
> EclipseSource GmbH
> Messerschmittstrasse 4
> 80992 Muenchen
>
> General Managers: Dr. Jonas Helming, Dr. Maximilian Koegel
> Registered Office: Messerschmittstrasse 4, 80992 Muenchen,
> Commercial Register Muenchen, HRB 191789
>


-- 
Stefan Dirix

Senior Software Architect
EclipseSource

Email: ***@***.***
Web: https://eclipsesource.com

EclipseSource GmbH
Messerschmittstrasse 4
80992 Muenchen

General Managers: Dr. Jonas Helming, Dr. Maximilian Koegel
Registered Office: Messerschmittstrasse 4, 80992 Muenchen,
Commercial Register Muenchen, HRB 191789

**Reply by ebugden** [reply]
Thank you Stefan!
